src/roulette.py

```diff
@@ -134,119 +134,99 @@ def __init__(self) -> None:
 
     def buildBins(self, wheel: Wheel) -> None:
         self.create_straight_bets(wheel)
+        self.create_horizontal_split_bets(wheel)
+        self.create_vertical_split_bets(wheel)
+        self.create_street_bets(wheel)
+        self.create_corner_bets(wheel)
+        self.create_line_bets(wheel)
+        self.generate_five_bets(wheel)
+        self.create_even_money_bets(wheel)
+        self.create_dozen_bets(wheel)
+        self.create_column_bets(wheel)
 
     @staticmethod
     def create_straight_bets(wheel: Wheel) -> None:
-        odd_for_straight_bet = 35
-        straight_bet_numbers = set(_ for _ in range(37))
+        straight_bet_odds = 35
+        straight_bet_numbers = {_ for _ in range(37)}
 
-        for i in straight_bet_numbers:
-            straight_bet = Outcome(str(i), odd_for_straight_bet)
-            wheel.addOutcome(i, straight_bet)
-        zero_zero_bet = Outcome("00", 35)
+        for number in straight_bet_numbers:
+            straight_bet = Outcome(str(number), straight_bet_odds)
+            wheel.addOutcome(number, straight_bet)
+        zero_zero_bet = Outcome("00", straight_bet_odds)
         bin_index_of_zero_zero_outcome = 37
         wheel.addOutcome(bin_index_of_zero_zero_outcome, zero_zero_bet)
 
     @staticmethod
     def create_horizontal_split_bets(wheel: Wheel) -> None:
-        odds_for_split_bet = 17
-
-        first_column_numbers = set()
-        for i in range(12):
-            first_column_numbers.add(i * 3 + 1)
-        for i in first_column_numbers:
-            column_one_two_split_bet = Outcome(str(i) + "-" + str(i + 1), odds_for_split_bet)
-            bin_indexes_for_horizontal_split_bets = i, i + 1
-            wheel.addOutcome(bin_indexes_for_horizontal_split_bets[0], column_one_two_split_bet)
-            wheel.addOutcome(bin_indexes_for_horizontal_split_bets[1], column_one_two_split_bet)
-
-        second_column_numbers = set()
-        for i in range(12):
-            second_column_numbers.add(i * 3 + 2)
-        for i in second_column_numbers:
-            column_two_three_split_bet = Outcome(str(i) + "-" + str(i + 1), odds_for_split_bet)
-            bin_indexes_for_horizontal_split_bets = i, i + 1
-            wheel.addOutcome(bin_indexes_for_horizontal_split_bets[0],
-                             column_two_three_split_bet)
-            wheel.addOutcome(bin_indexes_for_horizontal_split_bets[1],
-                             column_two_three_split_bet)
+        split_bet_odds = 17
+
+        split_bet_numbers = {number * 3 + 1 for number in range(12)} | {number * 3 + 2 for number
+                                                                        in range(12)}
+        for number in split_bet_numbers:
+            column_one_two_split_bet = Outcome(str(number) + "-" + str(number + 1),
```

Comment:
`column_one_two_split_bet` is no longer meaningful as `first_column_numbers` has been removed.

Reply:
Done. Thanks.